core/src/main/java/com/netflix/conductor/dao/MetadataDAO.java

```diff
@@ -103,7 +103,7 @@
 	 * @param name Name of the workflow
 	 * @param version version
 	 */
-	public void removeWorkflowDef(String name, int version);
+	public void removeWorkflowDef(String name, Integer version);
```

Comment:
Does the removeWorkflow also delete the workflows that were associated with the definition ? If so the Java doc needs to added for this.  And also what would happen when the Workflow definition or the version does not exist.

Try to add precise details to the java doc to avoid confusion, `@param name Name of the workflow ` can be changed to `@param name Name of the WorkflowDefinition that is intended to be deleted` 
and ` @param version version` to  `@param version version of the workflow definition that will be deleted`

Reply:
It just deletes the workflow definition. I will update the docs.

Reply:
It returns 404 incase workflow definition or the version does not exist.